src/api/vaults/types.ts

```diff
@@ -30,4 +30,5 @@ export type Vault = {
   pricePerFullShare: BigNumber;
   createdAt: number;
   chain: ApiChain;
+  totalSupply?: number;
```

Comment:
Gov vaults aren't the same as vaults, it might be worth having different types for each so it's easier to work with them on the api's side